docs/disabled-operator.md

```diff
@@ -0,0 +1,43 @@
+Cloud Credential Operator can be disabled prior to install in environments where the cloud IAM APIs are not reachable.
+
+  * Extract a release image: `oc adm release extract registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-10-23-103858`
+  * Edit 0000_50_cloud-credential-operator_01_operator_configmap.yaml and set disabled to "true".
+  * Examine all CredentialsRequests in the directory for your cloud platform. You will need to manually create secrets for each of these for the location and permissions specified in the request. However we will inject these secrets in the installer manifests rather than the release image, as we do not want to embed powerful cloud credentials in a published release image.
+  * Build a custom release image from what you extracted and modified: `oc adm release new --from-dir="." --to-image quay.io/dgoodwin/origin-release:latest`
+  * Generate installer manifests: `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/dgoodwin/origin-release:latest ./bin/openshift-install create manifests --dir=clusters/dgoodwin-aws`
+  * Add yaml files to the clusters manifests dir for each secret matching the CredentialsRequest in the release image for your cloud platform, as identified in the step above:
```

Comment:
I had to read this a couple of times before the step sunk in. This might make things a little clearer:
Add the yaml-formatted Secrets files to the cluster's manifests dir (eg clusters/dgoodwin-aws/manifests from the generate command above) for each secret...